test/config_cat_test.exs

```diff
@@ -13,7 +13,7 @@ defmodule ConfigCatTest do
   setup do
     feature = "FEATURE"
     value = "VALUE"
-    config = %{feature => %{"v" => value}}
+    config = %{"f" => %{feature => %{"v" => value}}}
```

Comment:
Should we use `Constants.feature_flags` (and `Constants.value` for the `"v"` part) here and elsewhere in the file now that they're extracted?

Reply:
Done 👍